src/main/java/org/usfirst/frc/team2473/robot/commands/ElevatorMove.java

```diff
@@ -0,0 +1,143 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package org.usfirst.frc.team2473.robot.commands;
+
+import org.usfirst.frc.team2473.robot.Robot;
+import org.usfirst.frc.team2473.robot.RobotMap;
+import org.usfirst.frc.team2473.robot.subsystems.Elevator.ElevatorPosition;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ * A class that enables accurate elevator mvoement to a given distance
+ * at a given power.
+ */
+public class ElevatorMove extends Command {
+	
+	/**
+	 * Lowest power elevator will run at towards the end of
+	 * the move.
+	 */
+	private static double SLOW_POWER = 0.1;
+	
+	/**
+	 * Number of inches to move by.
+	 */
+	private double moveByInches;
+	
+	/**
+	 * The number of ticks that the elevator should
+	 * be at if it has reached its goal.
+	 */
+	private double absoluteTickGoal;
+	
+	/**
+	 * The absolute number of ticks that the elevator
+	 * was at after the last call of execute.
+	 */
+	private int prevTicks;
+	
+	/**
+	 * Initial power to move the elevator at.
+	 */
+	private double power;
+	private double initialPower;
+
+	private ElevatorPosition pos;
+	
+	/**
+	 * Create a ElevatorMove object with given inch goal and power.
+	 * @param inches inches to move by
+	 * @param power initial power to move at
+	 */
+	public ElevatorMove(double inches, double power) {
+		requires(Robot.elevator);
+		
+		this.moveByInches = inches;
+		this.power = (inches < 0) ? -power : power;
```

Comment:
What if power is negative? If you are expecting power to be > 0, you need to:
1) Assert
2) Take the absolute value